grouper/email_util.py

```diff
@@ -256,3 +258,47 @@ def notify_edge_expiration(settings, session, edge):
         settings=settings,
         context=email_context,
     )
+
+
+def notify_nonauditor_flagged(settings, session, edge):
+    """Send notification that a nonauditor in an audited group has had their membership
+    set to expire.
+
+    Handles email notification and audit logging.
+
+    Args:
+        settings (Settings): Grouper Settings object for current run.
+        session (Session): Object for db session.
+        edge (GroupEdge): The expiring edge.
+    """
+    # TODO(rra): Arbitrarily use the first listed owner of the group from which membership expired
```

Comment:
I feel really uncomfortable about continuing this pattern, considering the number of places we've since duplicated it. Could you create a separate PR that does this the Right Way?